Standards/scs-xxxx-v1-provider-network-standard.md

```diff
@@ -0,0 +1,200 @@
+* `access_as_external` allows networks to be used as external gateway for virtual routers in the target projects. Such networks are in the following referred to as _external networks_.
+* `access_as_shared` allows networks to be attached directly to servers in the target projects. Such networks are in the following referred to as _shared networks_.
+
+The rules can be created with either a specific target project ID, or with a wild card (`*`) to target all projects.
+They can also overlap, allowing a network to be both external and shared to the same target projects.
+
+### Address Allocation and Routing
+
+CSPs can assign a subnet to a provider network to supply connected servers or virtual routers with public IP addresses, making them externally accessible.
+This works well for shared networks, where servers can be attached directly, although there is no quota option to limit the number of allocated addresses per project.
+
+Making servers in a project-internal network externally accessible through a virtual router is a bit more complicated, though.
+One option is for the user to create a subnet with a public IP range for the internal network, and then ask the CSP to configure a static route to the subnet via the gateway IP of a virtual router.
+This is cumbersome to set up manually, but can be automated with the `bgp` extension of the Network API, which is implemented by the `neutron-dynamic-routing` project [^bgp].
+For users, this takes the form of a CSP-managed shared subnet pool that they can create externally routable subnets from, limited by a per-project quota.
+
+For IPv6, there is also the option of prefix delegation, where a DHCPv6 server automatically assigns an IPv6 prefix to a subnet when it connects to the external provider network [^pd].
+However, this also means that ports in the subnet can lose their addresses and get assigned new ones if the subnet is removed from the external network and later reattached.
+The documentation at [^pd] still marks prefix delegation as an experimental feature in Neutron and notes low test coverage.
+
+For IPv4, OpenStack virtual routers support source NAT, allowing all servers in the internal subnet to access the external network with the gateway IP of the virtual router.
+They also support destination NAT in the form of floating IPs, addresses from the external network that can be mapped onto specific servers in the internal subnet to make them externally accessible.
+Floating IPs are allocated from a CSP-managed pool, and can be controlled by a per-project quota.
+There is also a set of API extensions that allow more fine grained port-forwarding, mapping different TCP or UDP ports of a floating IP to different internal IP addresses [^pf].
+
+### Port Security and Spoofing
+
+OpenStack networks have the flag `port_security_enabled`, that is set to true by default and can only be changed by it's owner.
```

Comment:
TIL `port_security_enabled` as a network-level flag, but it only controls the *default* for the port-level flag of the same name which has the functionality you describe.

I suggest a rewording:

> OpenStack ports have the flag `port_security_enabled`, that is set to true by default and can only be changed by the owner *of the corresponding network*. The default value of that flag is controlled by a `port_security_enabled` flag on the network.

Reply:
done